src/features/notification/screens/ParticipatingScreen.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+//neverdoco-opcoding
```

Comment:
just wonder, what this comment is for?

Reply:
Hmm.... I don't really know either. I didn't touch ParticipatingScreen at all :s